api/users/urls.py

```diff
@@ -5,7 +5,7 @@
 
 urlpatterns = [
     re_path(r'^reset_password/$', views.ResetPassword.as_view(), name=views.ResetPassword.view_name),
-    re_path(r'^external_login_comfirm_email/$', views.ExternalLoginConfirmEmailView.as_view(), name=views.ExternalLoginConfirmEmailView.view_name),
+    re_path(r'^external_login_confirm_email/$', views.ExternalLoginConfirmEmailView.as_view(), name=views.ExternalLoginConfirmEmailView.view_name),
```

Comment:
```suggestion
    re_path(r'^external_login_comfirm_email/$', views.ExternalLoginConfirmEmailView.as_view(), name=views.ExternalLoginConfirmEmailView.view_name), # Keep misspelled route here to avoid breaking existing links
    re_path(r'^external_login_confirm_email/$', views.ExternalLoginConfirmEmailView.as_view(), name=views.ExternalLoginConfirmEmailView.view_name),
```